### ISSUE TITLE
Dynamis Re-Entry Restriction

### ISSUE THREAD

**Issue by Acedffxi** [LegionDark/Issues#1378]
**Date & Time**:  2/21/2017

**FFXI Client Version (use `/ver`)**:  30161227_1

**Server's Expected Client Version matches, yes/no? (use `$ecv`)**:  Yes

**Character Name**:  Freelot

**Nation**:  Sandy

**Job(level)/Sub Job(level)**:  THF99/DNC49

**NPC or Monster or item Name**:  N/A

**Zone name**:  Dynamis 

**Coordinates (use `$where`)**:  N/A

**ffxiah.com link (for items issues only)**: N/A

**Multi-boxing? (multiple clients on same connection)**:  N/A

**Steps To Reproduce / any other info**: 
When trying to enter dynamis after exiting a previous one, it says "Unable to enter Dynamis for 14 vanadiel days."

**Comment by TeoTwawki** [comment]
Huh. I've never seen it say the time that way before..

Its normal to not be able to enter dynamis twice in one earth day day. for a long while it was broken and someone fixed a bunch of Dynamis issues in DSP recently.. Will investigate when I have time to see if this is just the normal entry wait being displayed differently or what.

**Comment by Acedffxi** [comment]
Yeah since joining this server there's never been a re-entry restriction. As of the patch today it's saying it now. Wasn't sure if it was intended or not so figured i'd post. :)

**Comment by Raelious** [comment]
It seems to let you reenter dynamis now without any 12hr reentry period.

**Comment by TeoTwawki** [comment]
That is strange because I haven't changed anything yet..

**Comment by Raelious** [comment]
I helped someone get a Dynamis-Windurst clear, killed the boss. Left and tried to reenter and it let me go back into Dynamis-Windurst with no timeout like people have been claiming.

**Comment by TeoTwawki** [comment]
Its (supposed to be) normal to be able to to re-enter if you have party members still inside. It's when the battle instance is "over with" that you can't go back in anymore. DSP has an inconsistent implementation of that feature - sometimes it works sometimes it doesn't. If the zone went completely empty and it still let you back in..That's a bug, and DSP will need told about it (again, ugh!).

**Comment by Raelious** [comment]
We fully cleared it, the person who needed the clear left. Then I left afterwards waited a few minutes and was able to reenter with all the mobs back up and attackable.

**Comment by TeoTwawki** [comment]
By fully cleared you mean beat it, or you mean all participants exited? The latter is the important part. ZOne doesn't reset till its got no players at all inside.

**Comment by Raelious** [comment]
We killed every single mob in the zone, killed the boss, then I killed all of the 100 coin statues after the boss died. I was the only one in the zone after the boss died. When I left there was no one in the zone and I waited about 2-3 minutes before attempting to re-enter Dynamis-Windurst, and it gave me the option to go inside, proceeded to and it re-spawned all of the mobs again, I killed the first 2 statues and mobs, exited and was able to reenter a third time with the mobs all back up again.
